aspnet/fundamentals/diagnostics.rst

```diff
@@ -91,13 +31,20 @@ The path for this page can be optionally specified in the call to ``UseRuntimeIn
 
 .. code-block:: c#
 
-	app.UseRuntimeInfoPage("/info");
+	if (env.IsDevelopment())
+	{
+	    app.UseRuntimeInfoPage("/info");
+	}
 
-As with ``UseDeveloperExceptionPage()``, it is a good idea to limit public access to diagnostic information about your application. As such, in our sample we are only implementing ``UseRuntimeInfoPage()`` when the EnvironmentName is set to Development.
+It is a good idea to limit public access to diagnostic information about your application. As such, you should only call ``UseRuntimeInfoPage()`` when the Environment is in Development.
```

Comment:
Diagnostic information should not be exposed in production. The code above prevents diagnostics outside the development environment.